pixeltable/catalog/table.py

```diff
@@ -859,13 +859,31 @@ def drop_column(self, column: str | ColumnRef, if_not_exists: Literal['error', '
                     f'{", ".join(c.name for c in dependent_user_cols)}'
                 )
 
-            _ = self._get_views(recursive=True, include_snapshots=False)
+            views = self._get_views(recursive=True, include_immutables=False)
+
+            # See if any view predicates depend on this column
+            dependent_views = []
```

Comment:
Should we break this out into a `_get_dependent_views()`?

Reply:
views = self._get_views(recursive=True, include_immutables=False)
we using this same list to check in external stores. I can create _get_dependent_views but then we will have 2 calls for self._get_views(recursive=True, include_immutables=False)

Marcel and I talked about adding this to catalog so we will always have column to view dependencies.